Networking/NetworkingTests/Remote/ProductVariationsRemoteTests.swift

```diff
@@ -117,9 +117,117 @@ final class ProductVariationsRemoteTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    // MARK: - Update ProductVariation
+
+    /// Verifies that updateProductVariation properly parses the `product-variation-update` sample response.
+    ///
+    func testUpdateProductVariationProperlyReturnsParsedProduct() {
+        // Given
+        let remote = ProductVariationsRemote(network: network)
+        let sampleProductVariationID: Int64 = 2783
+        network.simulateResponse(requestUrlSuffix: "products/\(sampleProductID)/variations/\(sampleProductVariationID)", filename: "product-variation-update")
+        let productVariation = sampleProductVariation(siteID: sampleSiteID, productID: sampleProductID, id: sampleProductVariationID)
+
+        // When
+        var updatedProductVariation: ProductVariation?
+        waitForExpectation { expectation in
+            remote.updateProductVariation(productVariation: productVariation) { result in
+                guard case let .success(productVariation) = result else {
+                    XCTFail("Unexpected result: \(result)")
+                    return
+                }
+                updatedProductVariation = productVariation
```

Comment:
I wonder if we should make a non-throwing version of `Result.get()` to make code like this shorter. 🤔 Something [like this](https://github.com/Alamofire/Alamofire/blob/becd9a729a37bdbef5bc39dc3c702b99f9e3d046/Source/Result%2BAlamofire.swift#L44-L48).

Reply:
ooh great idea, and I found a [native `Result.get()`](https://developer.apple.com/documentation/swift/result/3139397-get)! updated in https://github.com/woocommerce/woocommerce-ios/pull/2539/commits/0a372c05c4034812cefeb3ef6395e2d3df225cd5